hack/build-manifests.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+set -e
+
+PROJECT_ROOT="$(readlink -e $(dirname "$BASH_SOURCE[0]")/../)"
+
+# REPLACES_VERSION is the old CSV_VERSION
+#   if REPLACES_VERSION == CSV_VERSION it will be ignored
+REPLACES_VERSION="${REPLACES_VERSION:-0.0.1}"
+CSV_VERSION="${CSV_VERSION:-0.0.2}"
```

Comment:
Replaces version is for upgrading a CSV.  Have you already created 0.0.1? 

Note: If `REPLACES_VERSION` == `CSV_VERSION, the generator will ignore replaces version, so you probably want to set `CSV_VERSION` to `0.0.1` if it's your first CSV.

Reply:
Right. We are starting with 0.0.1.